package.json

```diff
@@ -42,35 +44,36 @@
     "react-slider": "1.0.1",
     "svgxuse": "1.2.6"
   },
+  "peerDependencies": {
+    "react": ">=18.2.0 <19.0.0",
+    "react-dom": ">=18.2.0 <19.0.0"
+  },
   "devDependencies": {
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "@hot-loader/react-dom": "^17.0.2",
```

Comment:
a bit too early
without demo we'll not be able to do other tasks

Reply:
Just reoredered